mcs/class/System/Mono.AppleTls/Enums.cs

```diff
@@ -9,8 +9,386 @@ namespace Mono.AppleTls {
 	// values are defined in Security.framework/Headers/SecBase.h 
 	enum SecStatusCode {
 		Success 							= 0,
-		DuplicateItem	 					= -25299,
```

Comment:
It is also already public in Xamarin.*.dll so we'll end up shipping it twice (lots of string metadata that only grows over time).

I understand it's nice for debugging the internals but it's not exposed. Maybe it could live under a `#if XXX` ?